### ISSUE TITLE
Move up to latest Tekton releases: Dashbhoard  and Webhook Extension

### ISSUE THREAD

**Issue by anneqm** [kabanero-io/kabanero-operator#441]
Tekton Dashboard with webhooks extension 0.5.1 will be available soon. 
Move up to Tekton Triggers 0.2.1and Tekton Pipelines 0.10.1 is being tracked in separate issue https://github.com/kabanero-io/kabanero-operator/issues/407 .

Note: Tekton Triggers 0.2.1 and Tekton Pipelines 0.10.1 are available from Tekton now, waiting for RH Openshift Releases to use with Kabanero.

**Comment by kaczyns** [comment]
Hi @anneqm - would you prefer we keep this issue over #407 which we already had open to do this work?

**Comment by anneqm** [comment]
@kaczyns that's perfect... shall we close this one, or modify the move up to the latest Dashboard and Webhook Extension work?

**Comment by marikaj123** [comment]
Anne, you and I can track this if you'd like. The team is aware of the Teckton dependencies. The team will make the updates as soon as they receive the links for the OpenShift pipeline 0.10.1 and Dashboard (0.5.1), Triggers (0.2.1).

**Comment by kaczyns** [comment]
@marikaj123 @anneqm 
I had a conversation with the pipelines team this morning.  We discussed options to try and make progress while we wait for the new Tekton versions to be picked up by OpenShift's version of the pipelines operator.  We considered creating a development cluster that had tekton pipelines 0.10.1 and triggers 0.2.1 manually installed (as opposed to using the operator).  This would require diverting resources away from the operator and pipelines teams, to build the environment, and both teams are already over-booked.  We discussed forking the OpenShift pipelines operator to pick just the fix that we need for webhooks, so that we can continue using pipelines version 0.8.x and triggers 0.1.x for Kabanero 0.6.0, but again we'd have to divert resources to do this.

We've decided to just wait for the OpenShift pipelines operator 0.10.1 to be available.  Technically we are blocked by the OpenShift pipelines operator being made available in github.com/operator-framework/community-operators, which is maintained by a group at Red Hat, so the label you picked `blockedBy-Tekton` is not really accurate.  

There was a status call today where someone (Raj?) mentioned being in closer communication with some of the OpenShift teams.  This issue would be an example of something that would benefit from that communication.  At this point, we are just waiting for the team at Red Hat to update the operator.  After that happens, the Kabanero Operator team will update Kabanero's operator catalog to pick it up, and then the Kabanero Pipelines team can start testing with that version and make any changes to the pipelines that are required by the new Tekton Pipelines release (0.10.1).

**Comment by anneqm** [comment]
@kaczyns The fix for webhooks also needs a triggers fix that is available in Triggers 0.2.1 so not sure we have the option to just pick the fix we need for webhooks.   We should have the dashboard release available any day now so we are also blocked by tekton until that's available.

**Comment by kaczyns** [comment]
I've removed the blocked-by openshift tag since we've received the openshift-pipelines operator.  Just waiting on Dashboard 0.5.1 now.

**Comment by anneqm** [comment]
Makes sense. Outlook for Dashboard 0.5.1 is early next week.

**Comment by anneqm** [comment]
@marikaj123 @kaczyns Tekton Dashboard and Webhooks Extension 0.5.1 is now available for the team to get started. https://github.com/tektoncd/dashboard/releases/tag/v0.5.1

**Comment by kaczyns** [comment]
Kabanero operator has Dashboard / webhooks extension v0.5.1 merged.
